schema/compose-spec.json

```diff
@@ -866,7 +867,34 @@
                   "path": {
                     "type": "string"
                   },
-                  "format": {
+                  "required": {
+                    "type": ["boolean", "string"],
+                    "default": true
+                  }
+                },
+                "required": [
+                  "path"
+                ]
+              }
+            ]
+          }
+        }
+      ]
+    },
+
+    "label_file": {
+      "oneOf": [
+        {"type": "string"},
+        {
```

Comment:
Sounds to me this is a premature feature to add support for `required` here. `label_file` should be declared at most a `string|array of string` for user convenience

Reply:
@ndeloof agree, fixed